examples/urls.py

```diff
@@ -1,12 +1,13 @@
-from django.conf.urls import include, url
+from django.conf.urls import include
+from django.urls import re_path
 from rest_framework.documentation import include_docs_urls
 
 from examples.rest import router
 from .views import index
 
 urlpatterns = [
-    url(r'^$', index, name='index'),
-    url(r'^', include(router.urls)),
-    url(r'^dynamicforms/', include('dynamicforms.urls')),
-    url(r'^api-docs/', include_docs_urls(title='Example API documentation')),
+    re_path(r'^$', index, name='index'),
```

Comment:
re_path je fallback za path in tam zamo za primere, ko s path RES ne moreš narest želenega. Vsaj tako pravi dokumentacija.